### ISSUE TITLE
Evaluation code

### ISSUE THREAD

**Issue by Danchik757** [kaviezhang/MeshMamba#4]
Thanks a lot for releasing the pretrained weights! I wrote my own evaluation code to reproduce the metrics (CC, SIM, KLD, SE) from the paper, but I’m getting noticeably different results. I may be missing some details in the evaluation process (e.g., normalization, averaging, or exact metric definitions).

Could you please share the evaluation code, or clarify the exact protocol you used to obtain the reported results? That would be very helpful for reproducing your work.

Thanks!

P.S. A side-by-side comparison for non-textured and textured meshes (paper vs. my results) is attached below.

<img width="720" height="489" alt="Image" src="https://github.com/user-attachments/assets/91571455-1c62-4309-996c-ef50b8e8e2a1" />

<img width="715" height="478" alt="Image" src="https://github.com/user-attachments/assets/fcbfac5c-f8e2-4af4-9b22-c6903203f8df" />

**Comment by kaviezhang** [comment]
Thank you for your reminder, and I apologize for the extra work caused by my oversight.
In fact, we missed an important dataset ordering file, `paths_dataset_rgb.txt`.
Since the division between the training and testing sets in the dataset has a significant impact on the results, all of our reported results are based on this specific ordering.
The file has now been uploaded to the directory.
Please modify  line 191 in  `train_mesh_Mamba.py` to use this file. Thx.

**Comment by Danchik757** [comment]
Good afternoon — following up on our earlier exchange. Thank you for being in touch. I’m still having trouble reproducing the evaluation results: the metrics improve only slightly, and I haven’t been able to match the values reported in the paper.

Could you please help with two items:
1. Do the model weights you shared reproduce the paper’s metrics on your side? (It’s possible the files I received aren’t the latest revision)
2. If possible, could you share the exact evaluation code or script you use?

Additionally, I noticed a discrepancy: the paper states an 80%/20% (train/test) split, but train_Mesh_Mamba.py appears to use a 60/40 % split by default. 

One more thing: your paper compares against the 3D saliency model SAL3D. I’ve already contacted the SAL3D authors; they indicated the repository isn’t public yet and didn’t provide a specific release timeline. If you know where the code can be obtained, or if you can suggest an alternative reference implementation, I’d be grateful.

Thank you again for your time.

**Comment by kaviezhang** [comment]
I'm so sorry about the mistake, the dataset split code should be '''
        dataset_paths["train"] = paths_dataset[:int(len(paths_dataset) * 0.8)]
        dataset_paths["test"] = paths_dataset[int(len(paths_dataset) * 0.8):] '''.

And about the SAL3D dataset, I will upload on the googlecloud sooner.
